deep_translator/glosbe.py

```diff
@@ -0,0 +1,105 @@
+"""
+google translator API
+"""
+
+__copyright__ = "Copyright (C) 2020 Nidhal Baccouri"
+
+from typing import List, Optional
+
+import requests
+
+from deep_translator.base import BaseTranslator
+from deep_translator.constants import BASE_URLS, GLOSBE_LANGUAGE_TO_CODE
+from deep_translator.exceptions import (
+    RequestError,
+    TooManyRequests,
+    ServerException,
+    TranslationNotFound
+)
+from deep_translator.validate import is_empty, is_input_valid, request_failed
+
+
+class GlosbeTranslator(BaseTranslator):
+    """
+    class that wraps functions, which use Glosbe Translate under the hood to translate text(s)
+    """
+
+    def __init__(
+        self,
+        source: str = "en",
+        target: str = "pl",
```

Comment:
why "pl" is default target?

Reply:
It was needed to keep a language as default for target, so I opened the Polish (pl) language and I kept it as it is. We can change it but there needs to be something default.